site/soc/software/charts/osh/openstack-compute-kit/neutron.yaml

```diff
@@ -42,6 +42,10 @@ data:
       interface:
         tunnel: {{ neutron_tunnel_device }}
```

Comment:
this defaults to `docker0`, right?

Reply:
nope, it defaults to null which means that osh will automagically get the proper interface:

```
 # If tunnel is null there is a fallback mechanism to search
 # for interface with routing using tunnel network cidr.
```